static/sass/styles.scss

```diff
@@ -48,10 +49,6 @@ $theme-default-nav: dark;
 @include vf-u-vertically-center;
 
 // Import site specific patterns and overrides
-@import "pattern_p-card";
-
-@include p-charmhub-card;
```

Comment:
Removed this as it will need a better solution when applying the platform icons.